### ISSUE TITLE
Enhancement: Send mail with attached Label

### ISSUE THREAD

**Issue by Kurt57** [shadimanna/dhl-logistic-services-for-woocommerce#215]
Would be greate if you can provide a functionallity if some Order Status is reached, that a mail to a given mail Adress is sent out, where the Label is attached.

Why?
im using HP Printer which has a mail Adress where i can send mails, which will be Printed by my HP Printer. So this way i can save some time to Create all manually and print manually.

in a Second step this would require do have some rules which time "warenpost" or "Paket" should be created. 

we have maintained all products with Weight and Sizes (width, length, ...) so if no product is present which is bigger than Warenpost requiments for me it would work.

**Comment by slieser** [comment]
Yes it would be great to automate the label printing! Sending it via email or at least support a url or api where I can download the label pdf file.
